app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
+_cache = {}
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def wrapper(*args) -> int:
```

Comment:
Are you sure you are returning int here ? And add pls also type hint for args :)

Reply:
And yes, we have 2 functions to decorate, and both functions return int